forte/processors/data_augment/algorithms/typo_replacement_op.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import random
+import json
+from typing import Tuple, Union, Dict, Any
+
+import requests
+from forte.data.ontology import Annotation
+from forte.processors.data_augment.algorithms.text_replacement_op import (
+    TextReplacementOp,
+)
+from forte.common.configuration import Config
+
+__all__ = [
+    "UniformTypoGenerator",
+    "TypoReplacementOp",
+]
+
+
+class UniformTypoGenerator:
+    r"""
+    A uniform generateor that generates a typo from a typo dictionary.
+
+    Args:
+        word: input word that needs to be replaced,
+        dict_path: the url or the path to the pre-defined typo json file.
+            The key is a word we want to replace. The value is a list
+            containing various typos of the corresponding key.
+            e.g. {"announced": ["anounced", "annouced"],
```

Comment:
We can formt the exapmle using `code-block`, see examples here:

https://github.com/asyml/forte/blob/master/forte/data/base_reader.py#L98

Reply:
I see. I added the JSON as code block and did a black check.